src/ontology/components/highlevelclass.tsv

```diff
@@ -1,38 +1,47 @@
+VBO:0400040	Golden hamster breed	VBO:0400000	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153			Vertebrate breed of the taxon Mesocricetus auratus. A vertebrate breed is a group of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) that distinguish it from other organisms of the same species, and/or for which cultural or geographical separation has led to the general acceptance of its separate identity. Breeds are formed through genetic isolation and either natural adaptation to the environment or selective breeding, or a combination of the two.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153		NCBITaxon:10036		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/		
+VBO:0400041	Actinopteri breed	VBO:0400000	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153			Vertebrate breed of the taxon Actinopteri comprised of all extant ray-finned fish minus the bichirs. A vertebrate breed is a group of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) that distinguish it from other organisms of the same species, and/or for which cultural or geographical separation has led to the general acceptance of its separate identity. Breeds are formed through genetic isolation and either natural adaptation to the environment or selective breeding, or a combination of the two.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153	https://en.wikipedia.org/wiki/Actinopteri	NCBITaxon:186623		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/		

```

Comment:
Could we call this term "fish breed"?

Reply:
yes, changed to NCBITaxon:32443 and term label to "fish breed"